silver/tests/commands/test_generate_docs.py

```diff
@@ -1305,7 +1307,7 @@ def test_canceled_subscription_with_trial_and_consumed_metered_features_draft(se
                                   generate_after=120, enabled=True,
                                   trial_period_days=7, amount=Decimal('200.00'),
                                   metered_features=[metered_feature])
-        start_date = dt.date(2015, 02, 01)
+        start_date = dt.date(2015, 0o2, 0o1)
```

Comment:
These weren't meant to be octals, sorry for that. The extra `0` was simply used for padding.